gadget/mkfs.go

```diff
@@ -17,7 +17,7 @@
  *
  */
 
-package internal
```

Comment:
we discussed and actually since this only uses gadget/quantity, which itself should be moved somewhere else, we think this package should be moved to `osutil/mkfs/mkfs.go` instead

Reply:
I don't think there should be any import cycles, since gadget/quantity just imports strutil, which shouldn't import any other packages from inside snapd since it's a *util package, but please let us know if you run into any import cycles by doing this move and we will help you resolve them since it probably will be non-trivial and we will have opinions on where to move things :smile: